jina/types/score/__init__.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+
+
+from ...excepts import BadNamedScoreType
+from ...proto import jina_pb2
+
+
+__all__ = ['NamedScore']
+
+
+class NamedScore:
+
+    def __init__(self, score: Optional[jina_pb2.NamedScoreProto] = None,
```

Comment:
docstring for this function, you can follow the style in other primitive data types